dashboard/config/routes.rb

```diff
@@ -6,8 +6,8 @@ module OPS
 end
 
 Dashboard::Application.routes.draw do
-  resources :featured_projects, only: [:create]
-  delete '/featured_projects/:project_id', controller: 'featured_projects', action: 'destroy_by_project_id'
+  put '/featured_projects/unfeature', to: 'featured_projects#unfeature'
+  post '/featured_projects/feature', to: 'featured_projects#feature'
```

Comment:
please change the second route from POST to PUT, here and in the js, because these routes are so similar in structure that it's a bit confusing to have one POST and one PUT. This is the best analysis of PUT vs POST I have been able to find: https://sookocheff.com/post/api/when-to-use-http-put-and-http-post/ . PUT seems like a closer match because you are updating a single resource, creating it if it does not exist, and the operation is idempotent.